data_environment.yml

```
@@ -0,0 +1,9 @@
+name: srw_app
```

Should the name associated with this be get_data?

```suggestion
name: get_data
```